configs/components/oifs/oifs43.env.yaml

```
@@ -47,19 +53,14 @@ computer:
             MAIN_LDFLAGS: '-openmp'
 
         levante:
+          iolibraries: system_libs
           add_export_vars:
             #IO_LIB_ROOT: "/work/ab0246/HPC_libraries/intel-oneapi-compilers/2022.0.1-gcc-11.2.0/openmpi/4.1.2-intel-2021.5.0/"
             #LD_LIBRARY_PATH: $IO_LIB_ROOT/lib:$LD_LIBRARY_PATH
 
-            SZIPROOT: "/sw/spack-levante/libaec-1.0.5-gij7yv"
-            HDF5ROOT: "/sw/spack-levante/hdf5-1.12.1-tvymb5"
-            HDF5_ROOT: $HDF5ROOT
-            NETCDFROOT: "/sw/spack-levante/netcdf-c-4.8.1-2k3cmu"
-            NETCDFFROOT: "/sw/spack-levante/netcdf-fortran-4.5.3-k6xq5g"
-            ECCODESROOT: "/work/ab0246/HPC_libraries/intel-oneapi-compilers/2022.0.1-gcc-11.2.0/openmpi/4.1.2-intel-2021.5.0"
 
             PATH: "$ECCODESROOT/bin:$PATH"
-            LD_LIBRARY_PATH[(3)]: "$ECCODESROOT/lib:$LD_LIBRARY_PATH"
+            LD_LIBRARY_PATH[(3)]: "$LD_LIBRARY_PATH:$ECCODESROOT/lib"
 
```

Just to confirm that this is intentional. In this way `ECCODESROOT/lib` has less preference than the libs defined in `LD_LIBRARY_PATH`, right? Is this the intended behavior? If not you can reverted with the following suggestion.

```suggestion
            LD_LIBRARY_PATH[(3)]: "$ECCODESROOT/lib:$LD_LIBRARY_PATH"
```

---

I forgot exactly why this was done. Especially for 43 it does not seem to make sense to me. I reverted back.